### PR TITLE
zh-translation: /news/releases/1.0.x/announcing-1.0.4/index.md #1531

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1531](https://github.com/servicemesher/istio-official-translation/issues/1531)

### CODE REVIEW

File: content/zh/news/releases/1.0.x/announcing-1.0.4/index.md [hunk 1]
```diff
@@ -12,24 +12,24 @@ aliases:
...
-  Once Pilot enters a deadlock, it exhibits continuous memory growth eventually running out of memory.
+- 使用 [`istioctl proxy-status`](/zh/docs/reference/commands/istioctl/#istioctl-proxy-status) 来获取代理同步状态时，可能会导致 Pilot 死锁。
+  临时的解决方法是不使用 `istioctl proxy-status`。
+  一旦 Pilot 进入死锁状态，它将表现出持续的内存增长，最终耗尽内存。
 
-## Networking
+## 网络{#networking}
 
-- Fixed the lack of removal of stale endpoints causing `503` errors.
+- 修复了缺少 removal 导致 503 错误的远古 endpoint bug。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.4/index.md]:
```suggestion
- 修复了过期 endpoint 漏删导致 503 错误的 bug。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.4/index.md [hunk 2]
```diff
@@ -12,24 +12,24 @@ aliases:
...
 
-- Fixed sidecar injection when a pod label contains a `/`.
+- 修复了 Pod 标签包含 `/` 时 sidecar 的注入 bug。
 
-## Policy and telemetry
+## 策略和遥测{#policy-and-telemetry}
 
-- Fixed occasional data corruption problem with out-of-process Mixer adapters leading to incorrect behavior.
+- 修复了进程外 Mixer 适配器造成的不正确行为，偶尔导致数据损坏的 bug。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.4/index.md]:
```suggestion
- 修复了进程外 Mixer 适配器偶尔的数据损坏问题导致的不正确行为。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.4/index.md (hunk 1 above)
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.4/index.md]: 已修改

File: content/zh/news/releases/1.0.x/announcing-1.0.4/index.md (hunk 2 above)
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.4/index.md]: 已修改，感谢您耐心的 review

image

**Pull request merged**